inference/wasm/tests/test-cases/translate-html-multi-paragraph-no-pivot.test.mjs

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { describe } from "vitest";
+import { runTranslationTest } from "./test-cases/shared.mjs";
+
+/**
+ * This file tests the WASM bindings for non-pivot translation requests
+ * that contain HTML tags within the source text, extended to multiple paragraphs.
+ */
+const testCases = [
+  {
+    sourceLanguage: "es",
```

Comment:
Thought: Just as an FYI, this style of test won't work with Jest's `matchInlineSnapshot`, since it's iterating over data, rather than including the assertions inline inside of a `matchInlineSnapshot` call. No action is needed, I just wanted to point it out.